docs/blog/2024-07-25-nx-19-5-update.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Nx 19.5 Update!!
+slug: 'nx-19-5-update'
+authors: ['Zack DeRose']
+cover_image: '/blog/images/2024-07-25/nx-19-5-thumbnail.png'
+tags: [nx, release]
+---
+
+## NEW: Pattern Support for `targetDefaults`
+
+We've added Pattern Support for `targetDefaults` in your `nx.json` file.
+
+Generally speaking, `targetDefaults` are a way for you to specify default configurations for your workspace. For example, if for your workspace, your `test` tasks always depend on your `build` task to have run before the `test`s can run, you'd add this as a `targetDefault`:
+
+```json
+{
+  "targetDefaults": {
+    "test": {
+      "dependsOn": ["build"]
+    }
+  }
+}
+```
+
+By adding pattern matching to `targetDefault`s, we can support a wider range of target names, which fits in really well with recent enhancements from Project Crystal, like task atomization. For example, the `@nx/playwright` plugin will automatically add in a task with the name `e2e-ci--${testFileName}` for every playwright spec file in your project. With this new pattern support in `targetDefaults`, we can define default behavior for all these tasks.
+
+The following will set all such tasks to depend on their `build` task:
+
+```json
+{
+  "targetDefaults": {
+    "e2e-ci--**/*": {
+      "dependsOn": ["build"]
+    }
+  }
+}
+```
+
+## NEW: Individual Targets Can Now Opt Out of Parallelism
+
+The ability to "Atomize" larger tasks by automatically breaking them up into smaller pieces is a really cool enhancement that is enabled by Nx Project Crystal. Especially when combined with the ability to distribute tasks across many machines by using Distributed Task Execution from Nx Agents.
```

Comment:
The main purpose of this was to allow e2e-ci tests to not have to specify `--parallel 1` and have a 2nd line. So we should build the section around that feature in the first place, plugging the [e2e test splitting](https://nx.dev/ci/features/split-e2e-tasks) and then build up how this new feature of opting out of parallelism enhances it.